generation/realism_diversity_metrics/realism_diversity_metrics.ipynb

```diff
@@ -0,0 +1,811 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    unet_url = "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/tutorial_generation_unet.pth"
</pre><p>To avoid confusion with maisi later, please help modify the weight path to include "2d_ldm".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1771/discussion/#comment-1718243923'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>